includes/API.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_5_4\SV_WC_API_Base;
+use SkyVerge\WooCommerce\PluginFramework\v5_5_4\SV_WC_API_Request;
+use SkyVerge\WooCommerce\PluginFramework\v5_5_4\SV_WC_Plugin;
+
+defined( 'ABSPATH' ) or exit;
+
+/**
+ * API handler.
+ *
+ * @since 2.0.0-dev.1
+ */
+class API extends SV_WC_API_Base {
+
+
+	/** @var string the configured access token */
+	protected $access_token;
+
+
+	/**
+	 * Gets a Page object from Facebook.
+	 *
+	 * @since 2.0.0-dev.1
+	 *
+	 * @param $page_id page ID
+	 * @return
+	 */
+	public function get_page( $page_id ) {
+
+		// TODO: Implement get_page() method.
+	}
+
+
+	/**
+	 * Uses the Catalog Batch API to update or remove items from catalog.
+	 *
+	 * @see Sync::create_or_update_products()
+	 *
+	 * @since 2.0.0-dev.1
+	 *
+	 * @param array $requests array of prefixed product IDs to create, update or remove
```

Comment:
```suggestion
	 * @param string[] $requests array of prefixed product IDs to create, update or 
```

then it's an array of strings if this is not an associative array (can update later)